### ISSUE TITLE
rollup --watch fails with no error if it tries to watch more files than max_user_watches setting.

### ISSUE THREAD

**Issue by cs2dsb** [rollup/rollup#2818]
- Rollup Version: 1.10.0
- Operating System (or Browser): Ubuntu 18.04.2 LTS
- Node Version: v8.10.0

### How Do We Reproduce?

Set fs.inotify.max_user_watches lower than the number of watched files in project.

### Expected Behavior

A descriptive error or fallback behavior (cargo watch for example tells you what to set max_user_watches to and falls back to polling).

### Actual Behavior

Exits with no message and exit code 1.

Default for Ubuntu 18.04 appears to be 500,000 and in my newly created react + semantic-ui-react exceeds this limit.

**Comment by lukastaegert** [comment]
Is this still an issue with Rollup 2?

**Comment by iamlouk** [comment]
@lukastaegert Yes it is, I just wanted to open an Issue about this. It is really confusing because there is no error message at all.

Thank you for your work!

Rollup Version: 2.3.4
OS: Manjaro with the latest updates, Kernel: Linux 5.11.16
Node Version: v16.0.0
